source/tutorials/index.rst

```diff
@@ -13,17 +13,15 @@ Explore our tutorials to discover what you can do with |CL|!
    flatpak/flatpak
    machine-learning/machine-learning
    docker/docker
-   azure
    hadoop
    fmv
-   aws-web/aws-web
    smb/smb*
    spark
    kata
-   kata_migration
    kubernetes/kubernetes*
    yubikey-u2f
    nvidia
    nvidia-cuda
    redis
-   tutorial-proxy
+   proxy
+   kata_migration
```

Comment:
Suggest ordering alphabetically to make it easier for readers to find if they know what they're looking for. 
Can we remove the `kata_migration` document altogether? See my note in the GH issue.